lib/modules/manager/gradle/parser/common.ts

```diff
@@ -1,4 +1,5 @@
-import { lexer, query as q } from 'good-enough-parser';
+import { lexer, parser, query as q } from 'good-enough-parser';
+import type { SeqBuilder } from 'good-enough-parser/dist/cjs/query/builder';
```

Comment:
@zharinov type should be exported

Reply:
No, these classes are implementation detail. Need another way 🤔

Reply:
PR-for-PR: https://github.com/Churro/renovate/pull/1

Reply:
Thanks!